Formula/n/notation.rb

```diff
@@ -0,0 +1,30 @@
+class Notation < Formula
+  desc "CLI tool to sign and verify OCI artifacts and container images"
+  homepage "https://notaryproject.dev/"
+  url "https://github.com/notaryproject/notation/archive/refs/tags/v1.0.0.tar.gz"
+  sha256 "c7e3a154cb132d17c72c9954e2c2144b5dd1781af69e9bfe25ec29916e2f01be"
+  license "Apache-2.0"
+  head "https://github.com/notaryproject/notation.git", branch: "main"
+
+  depends_on "go" => :build
+
+  def install
+    project = "github.com/notaryproject/notation"
+    ldflags = %W[
+      -s -w
+      -X #{project}/internal/version.Version=v#{version}
+      -X #{project}/internal/version.GitCommit=#{tap.user}
```

Comment:
I'm not able to find the documentation for `#{tap.user}`. What's the value of it? May I have the link to the documentation?

Reply:
Don't know why but it's value is `Homebrew`, and value of `tap` is `homebrew/core`. Looks like it is the repository's or tap's name.

Reply:
We can just remove line 16. If `GitCommit` is empty, `notation version` will show
```
notation version
Notation - a tool to sign and verify artifacts.

Version:     v1.0.0+Homebrew
Go version:  go1.21.0
```
without `Git commit:`.

See code: https://github.com/notaryproject/notation/blob/v1.0.0/cmd/notation/version.go#L42-L44

Reply:
But, `Utils.git_short_head` provides the git commit. So why not use it?